packages/testcontainers/src/docker-compose-environment/docker-compose-environment.test.ts

```diff
@@ -130,7 +129,7 @@ describe("DockerComposeEnvironment", () => {
           .withWaitStrategy(await composeContainerName("container"), Wait.forHealthCheck())
           .withStartupTimeout(0)
           .up()
-      ).rejects.toThrowError(`Health check not healthy after 0ms`);
+      ).rejects.toThrow(`Health check not healthy after 0ms`);
```

Comment:
What's the reason for changing this? Ideally an error should be thrown so we have a stack trace, instead of just a string

Reply:
![image](https://github.com/testcontainers/testcontainers-node/assets/1389199/4845daeb-fab6-4d6a-b0f4-8f47df573c54)

`toThrowError` in deprecated and replaced with `toThrow`. Signature is same and `toThrow` is mentioned, so I'd assume it functions in identically (tho I didn't check the deprecation notes).

Reply:
Oh I didn't know it had been deprecated, then please keep it, thank you